code/go/internal/validator/folder_item_content.go

```diff
@@ -9,59 +9,122 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	"mime"
+	"os"
 
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v3"
+
+	ve "github.com/elastic/package-spec/code/go/internal/errors"
+	"github.com/elastic/package-spec/code/go/internal/spectypes"
+)
+
+const (
+	maxFileSize              = 150 * spectypes.MegaByte
+	maxConfigurationFileSize = 5 * spectypes.MegaByte
```

Comment:
I'm wondering if there shouldn't be a general YAML file for default settings like this.

Reply:
Yep, it can make sense to make them part of the folder spec, this would allow to define these settings per directory too. I will give it a try as part of the implementation of the global limits.